.github/workflows/test.yaml

```diff
@@ -0,0 +1,33 @@
+name: CI
+on: pull_request
+
+jobs:
+  ci:
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.8, 3.9]
+        poetry-version: [1.1.11]
+        os: [ubuntu-18.04, macos-latest, windows-latest]
+    runs-on: ${{ matrix.os }}
+    services:
+      rabbitmq-streaming:
+        image: pivotalrabbitmq/rabbitmq-stream
+        env:
+          RABBITMQ_SERVER_ADDITIONAL_ERL_ARGS: "-rabbitmq_stream  advertised_host localhost"
+        ports:
+          - 5552:5552
+          - 15672:15672
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Run image
+        uses: abatilo/actions-poetry@v2.0.0
+        with:
+          poetry-version: ${{ matrix.poetry-version }}
+      - name: poetry install
+        run: poetry install
+      - name: poetry run pytest
+        run: poetry run pytest
```

Comment:
Never used actions before, but if I understand correctly rabbit's host will be mapped to the service name - `rabbitmq-streaming` so I guess we need to invoke pytest with `pytest --rmq-host rabbitmq-streaming`

Is there a way to run this before merging? Or perhaps we can debug this in a next PR

Reply:
It uses `localhost:5552`.  We use it in all [the clients](https://github.com/rabbitmq/rabbitmq-stream-go-client/blob/main/.github/workflows/test_and_publish_image.yml#L8-L24)